Gemfile

```diff
@@ -27,6 +27,7 @@ gem 'turbolinks'
 
 # View
 gem 'simple_form'
+gem 'enumerize'
```

Comment:
I rebased both of my branches on top of `master`. And that Gemfile looks the same to me :)